system-variables.md

```diff
@@ -127,6 +127,12 @@ mysql> SELECT * FROM t1;
 - This variable indicates the location where data is stored. This location can be a local path or point to a PD server if the data is stored on TiKV.
 - A value in the format of `ip_address:port` indicates the PD server that TiDB connects to on startup.
 
+### default_authentication_plugin
+
+- Scope: GLOBAL
+- Default value: `mysql_native_password`
+- This variable sets the authentication method the server advertises during connection establishment.
+
```

Comment:
Are the value options of this variable documented in `## Authentication plugin status`?

Reply:
Yes. Updated the text to add a link

Reply:
Yes, only `mysql_native_password` and `caching_sha2_password` are allowed and TLS client authentication works differently: https://docs.pingcap.com/tidb/stable/enable-tls-between-clients-and-servers#enable-authentication